### DISCUSSION TITLE
URL of latest ApsimX version

### DISCUSSION THREAD

**Discussion by youngr1960** [APSIMInitiative/ApsimX#8801] [General]
I use the link http://apsimdev.apsim.info/APSIM.Builds.Service/Builds.svc/GetURLOfLatestVersion?operatingSystem=Debian to get the latest version number of ApsimX, which is then used to download the latest version of the ApsimX debian package. This link doesn't seem to be available anymore as it returns "...could not be found". Does anybody know if this link is still available or has changed? Thankyou

**Comment by hol353** [comment]
Apologies, the endpoint has moved. Try this:
```
curl -X POST https://builds.apsim.info/api/nextgen/list?n=1

[{"releaseDate":"2024-04-02T22:15:04","issue":8782,"title":"Need a manager to manipulate species diversity in a pasture simulation","downloadLinkDebian":"https://builds.apsim.info/api/nextgen/download/7437/Linux","downloadLinkWindows":"https://builds.apsim.info/api/nextgen/download/7437/Windows","downloadLinkMacOS":"https://builds.apsim.info/api/nextgen/download/7437/MacOS","infoUrl":"https://github.com/APSIMInitiative/ApsimX/issues/8782","version":"2024.04.7437.0","revision":7437}]
```

This will give you the debian install link.
